policyengine_us/variables/gov/local/ca/la/dwp/ez_save/la_ez_save_eligible.py

```diff
@@ -0,0 +1,22 @@
+from policyengine_us.model_api import *
+
+
+class la_ez_save_eligible(Variable):
+    value_type = bool
+    entity = SPMUnit
```

Comment:
I'd expect since it's a utility
```suggestion
    entity = Household
```